kahuna/public/stylesheets/main.css

```diff
@@ -308,9 +308,24 @@ a:focus {
     display: inline-block;
 }
 
-/* ==========================================================================
-   Generics
-   ========================================================================== */
+.side-padded {
+    padding: 0 10px;
+}
+
+.clickable:hover {
```

Comment:
I don't mind, I just wondered if `clickable` was the right word - but I suppose something can look clickable. :+1: